docs/source/guide/zne-3-options.md

```diff
@@ -225,14 +225,18 @@ $$G \longrightarrow I G$$
 
 Here, $G$ is a single circuit layer containing individual that can be performed simultaneously.
 
-To use this method, call {func}`.zne.scaling.insert_id_layers()` with both a circuit to scale, and a scale factor.
+To use this method, call {func}`.zne.scaling.identity_insertion.insert_id_layers()` with both a circuit to scale, and a scale factor.
```

Comment:
Since `insert_id_layers` is imported in [`zne/scaling/__init__.py`](https://github.com/unitaryfund/mitiq/blob/master/mitiq/zne/scaling/__init__.py), this change isn't totally necessary. Is there a reason you prefer this? I personally prefer a shorter function/module chain.

Reply:
Yes, I understand this and I would have used `zne.scaling.insert_id_layers()` as well. I noticed that in the docs, it was not linking to the API docs. It is not in blue. 

![image](https://user-images.githubusercontent.com/66048318/229244878-b0ae85ee-85c4-44d3-8a4f-a3f223673104.png)

Reply:
Ahhh okay sphinx must not be able to find it otherwise. Thanks for checking.